modules/nf-core/seqkit/seq/tests/main.nf.test

```diff
@@ -26,7 +26,6 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(process.out).match() },
-                { assert snapshot(process.out.versions).match("versions") }
```

Comment:
Thanks!  This is what I was hoping for, no longer considering checksums on the versions file.

Reply:
Well, the versions is being checked in the line above, as it is part of `process.out`